src/app/welcome/welcome.component.html

```diff
@@ -66,4 +67,21 @@ <h1>Welcome to the EFIS editor!</h1>
       Be sure to download a copy of your data and keep it somewhere safe.
     </div>
   </div>
+
+  <div class="callout important">
+    <div class="callout-title">IMPORTANT:</div>
+    <div>
+      Not all formats can represent entered items in the same way! Do check that the checklists are complete and appear
+      as you expected when converting between formats.<br />
+      Specifically, the ForeFlight format supports:
+      <ul>
+        <li>Only 3 pre-set group category types (Normal, Abnormal, Emergency)</li>
+        <li>
+          Title, Challenge, Challenge/Response and Note to Challenge/Response (exactly 1 level indent) checklist item
+          types
+        </li>
+      </ul>
+      Other items will be silently ignored when exporting the checklist file!
```

Comment:
Can we instead convert them to Challenge type? It's probably better to lose formatting than to lose the whole item.

Reply:
I implemented full mapping in ac46d25 and changed the note in e8d3999. I think it's still useful to point out that different formats can represent checklists in possibly unexpected ways.